prov/util/src/util_ep.c

```diff
@@ -190,6 +191,20 @@ int ofi_ep_bind(struct util_ep *util_ep, struct fid *fid, uint64_t flags)
 	return -FI_EINVAL;
 }
 
+static inline int util_coll_init_cid_mask(struct bitmask *mask)
+{
+	int err = ofi_bitmask_init(mask, OFI_CONTEXT_ID_SIZE);
```

Comment:
Can we get a better name than CONTEXT_ID_SIZE?  Does this represent the maximum number of collectives that we can perform?  that can be active?

Reply:
currently, it's the max we can perform.  I think when a collective mc is closed, we could reacquire that context id and it would become the max which can be active.